### ISSUE TITLE
+v2.0

### ISSUE THREAD

**Issue by ue71603** [NeTEx-CEN/NeTEx#636]
Note that in the past we have added an indication in the XML documentation as to which release an enhancement was added (e.g. +v1.1, +v2/0) as this makes it easy to see which attributes cannot be used with an earlier version and what has changed without resorting to github etc, Would it not be a good idea to continue doing so?

**Comment by Aurige** [comment]
that's right @ue71603, that's useful, but that may not be an easy task to add it now... I don't see how to automatically add +v2 to all the changes that were already validated. Do you see an easy way to do it ?

**Comment by ue71603** [comment]
@Aurige : I think we can't do it automatically.  But we can:
- start doing it
- perhaps when @trurlurl  helps with the documentation we can add it in a PR that updates it in parallel (like we will do with the search for typos).

**Comment by Aurige** [comment]
Fine for me

**Comment by Aurige** [comment]
Group decision: yes try to bring this back ! Only for new attribute/element (not on updates)

**Comment by Aurige** [comment]
Update XSD while updating the document (@trurlurl )

**Comment by Aurige** [comment]
Also update the NeTEx version in file's header (only if there are some changes)

**Comment by trurlurl** [comment]
Quite some work has been done in #762, #764, #781, #799. This improves the situation at the level of the element annotations and the documentation, but to a lesser extent in the headers. The headers don't seem to be a very reliable source of information to me - and wouln't they be somehow redundant (if they were up to date) to the changelog? 
@Aurige Shall we close this issue? Some action or reminder needed regarding headers?

**Comment by Aurige** [comment]
I thinks that we can close it since a quite huge amount a work has been done.
I agree that headers are not always as useful as they should be, but this can be an independent action
